README_en.md

```diff
@@ -0,0 +1,155 @@
+**Quick start PaddleClas in 30 minutes**: https://paddleclas.readthedocs.io/zh_CN/latest/tutorials/quick_start.html
+
+## Introduction
+
+PaddleClas is a tool set for image classification tasks prepared for the industry and academia. It helps users train better computer vision models and apply in real scenarios.
+
+<div align="center">
+    <img src="./docs/images/main_features_s_en.png" width="700">
+</div>
+
+## Rich model zoo
+
+Based on ImageNet1k dataset, PaddleClas provides a brief introduction to 23 series of imaget classification networks such as ResNet, ResNet_vd, Res2Net, HRNet, and MobileNetV3 besides their reproduction configurations and training techniques. At the same time, the corresponding 117 image classification pretrained models are also open source. The GPU inference time of the server-side model is evaluated based on TensorRT. The CPU inference time and the mobile-side model storage size are evaluated on the Snapdragon 855 (SD855). For more detailed information of the supported pretrained models and their download links, please refer to [**models introduction tutorial**](https://paddleclas.readthedocs.io/zh_CN/latest/models/models_intro.html).
+
+
+<div align="center">
+    <img src="./docs/images/models/V100_benchmark/v100.fp32.bs1.main_fps_top1_s.jpg" width="700">
+</div>
+
+The above figure shows some of the latest server-side pretrained models. It can be seen from the figure that when using V100 GPU with FP32 and TensorRT, the `Top1` accuracy of the ResNet50_vd_ssld pretrained model on ImageNet1k-val dataset is **82.4%** and that of ResNet101_vd_ssld pretrained model is 83.7%. These pretained models are obtained from  SSLD knowledge distillation solution provided by PaddleClas. The marks of the same color and symbol in the figure represent models of different model sizes in the same series. For the introduction of different models, FLOPS, Params and detailed GPU inference time (including the infecne speed of T4 GPU with different batch size), please refer to the documentation tutorial formal details: [https://paddleclas.readthedocs.io/zh_CN/latest/models/models_intro.html](https://paddleclas.readthedocs.io/zh_CN/latest/models/models_intro.html)
+
+
+<div align="center">
+<img
+src="./docs/images/models/mobile_arm_top1.png" width="700">
+</div>
+
+
+The above figure shows performance of some commonly used mobile-side models, including MobileNetV1, MobileNetV2, MobileNetV3 and ShuffleNetV2 series. The inferece time is tested on Snapdragon 855 (SD855) with the batch size set as 1. the `Top1` accuracy of the MV3_large_x1_0_ssld, MV3_small_x1_0_ssld, MV1_ssld and MV2_ssld pretrained model on ImageNet1k-val dataset are 79%, 71.3%, 76.74%, 77.89%, respectively (M is short for MobileNet). MV3_large_x1_0_ssld_int8 is a quantizatied pretrained model for MV3_large_x1_0. More details about the mobile-side models can be seen in [**models introduction tutorial**](https://paddleclas.readthedocs.io/zh_CN/latest/models/models_intro.html)
```

Comment:
performance -> the performance
inferece -> inference

Reply:
fixed